docs/usage/files.md

```diff
@@ -761,3 +762,57 @@ for instance in file_metadata:
 ```
 
 [get_all_metadata]: https://box-python-sdk.readthedocs.io/en/latest/boxsdk.object.html#boxsdk.object.item.Item.get_all_metadata
+
+Set a Classification
```

Comment:
It might be good to note that this functionality is only available to customers with Governance